### PR TITLE
Revert to raster level meters in LateNight and Tango to fix severe GUI lag on macOS

### PR DESCRIPTION
**Pull request by Be-ing**
Rendering SVGs for the meters causes the whole GUI to become severely laggy on older Macs:
https://bugs.launchpad.net/mixxx/+bug/1759433

### CODE REVIEW

**Comment by Be-ing** [comment]
I just tested with 250% scaling on my 3840 x 2160 screen with GNU/Linux and the meters still look good to me. I don't notice any blurriness.

File: res/skins/Tango/vumeter_floating.xml [hunk 1]
```diff
@@ -21,15 +25,15 @@ Variables:
...
+                <PathBack>skin:/graphics/vumeter_floating_clipping.png</PathBack>
                 <Connection>
                   <ConfigKey><Variable name="group"/>,PeakIndicatorL</ConfigKey>
                 </Connection>
               </StatusLight>
               <VuMeter>
-                <PathVu>graphics/vumeter_floating_over.svg</PathVu>
-                <PathBack>graphics/vumeter_floating.svg</PathBack>
+                <PathVu>skin:/graphics/vumeter_floating_over.png</PathVu>
+                <PathBack>skin:/graphics/vumeter_floating.</PathBack>
```
Comment (ronso0) [inline res/skins/Tango/vumeter_floating.xml]: PNG extension is missing

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by ronso0** [comment]
... the rest LGTM

**Comment by ronso0** [comment]
I just noticed the Aux mute/play button icons in LateNight is reverted.
@Be-ing could you fix this here as well? Just swap the `<Number>`s at https://github.com/mixxxdj/mixxx/blob/master/res/skins/LateNight/aux_unit.xml#L42

File: res/skins/Tango/vumeter_floating.xml (hunk 1 above)
Comment (Be-ing) [reply res/skins/Tango/vumeter_floating.xml]: Good catch, thanks

**Comment by Be-ing** [comment]
FYI the SVG files are still in the Git repository. I just rendered them to PNG.

**Comment by Be-ing** [comment]
I double checked this on the Macbook Air that I am borrowing and it works. :tada:

**Comment by Be-ing** [comment]
> I just noticed the Aux mute/play button icons in LateNight is reverted.

Please open another PR for that.

**Pull request merged**